IopHomeNode.proto3

```diff
@@ -248,6 +248,12 @@ message ConversationResponse {
 //
 // An administrative message to test whether the peer is there and responds to requests.
 //
+// Each identity client is required to send a message not more than 60 seconds after the connection 
```

Comment:
These numbers are according to @furszy - his reasoning was as you wrote - the mobile devices are less reliable in closing the connections. It has been argued that sending a couple of packets every minute is not a big deal.

These numbers do affect the resources consumed on the node. If a malicious client opens a connection and does not close it, there is a one less legitimate client that the node can effectively serve (so this is a minor DoS prevention as well). There are resources consumed by the open TCP connection itself as well as some resources that the node's implementation uses for each connected client. If there are just tens of those connections, it is not a big deal. If there are thousands, it is a problem.

If these numbers are not reasonable, please discuss with @furszy and upon mutual agreement, please suggest a better alternative.